src/vos/vos_gc.c

```diff
@@ -182,8 +188,24 @@ gc_drain_key(struct vos_gc *gc, struct vos_pool *pool, daos_handle_t coh,
 		pool->vp_gc_stat.gs_recxs += creds;
 	return 0;
 }
-
-/**
+// /* flat key*/
+// static int
+// gc_free_dkey(struct vos_gc *gc, struct vos_pool *pool, struct vos_gc_item *item)
+// {
+// 	struct vos_krec_df *key = umem_off2ptr(&pool->vp_umm, item->it_addr);
+
+// 	D_ASSERT(key->kr_bmap & KREC_BF_DKEY);
+// 	/*
+// 	flat key: coh = DAOS_HDL_INVAL!!!!
+// 	*/
```

Comment:
(style)  please, no space before tabs
(style)  Block comments use a trailing */ on a separate line
(style)  do not use C99 // comments